registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java

```diff
@@ -1279,6 +1279,14 @@ public enum PlatformErrorMessages {
 	
 	RPR_DEMO_API_RESOUCE_ACCESS_FAILED(PlatformConstants.RPR_DEMO_DEDUPE_MODULE + "002",
 			"Unable to Access API Resource"),
+	RPR_DEMO_BIOMETRIC_RECORD_VALIDATION_FAILED(PlatformConstants.RPR_DEMO_DEDUPE_MODULE + "003",
+			"Biometric record validation error"),
+
+	RPR_DEMO_POTENTIAL_SENDING_FOR_MANUAL(PlatformConstants.RPR_DEMO_DEDUPE_MODULE + "004",
+			"Potential Demo Match was Found and Biometric record validation failed. Hence sending to manual adjudication"),
+
+	RPR_DEMO_POTENTIAL_PACKET_REJECTED(PlatformConstants.RPR_DEMO_DEDUPE_MODULE + "004",
+			"Potential Demo Match was Found and Biometric record validation failed. Hende demodedupe Rejected"),

```

Comment:
hence spell check and lets tell packet is rejected not demodedupe rejected

Reply:
fixed